### ISSUE TITLE
Df import Syntax errors

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#4]
The following two examples produce import errors:

**First Error:** Syntax error <unknown>:13640:2 extraneous input 'FIELD-MISC11' expecting ...
```
ADD FIELD "testField" OF "XX123" AS decimal 
  DESCRIPTION "header"
  FORMAT "->>>9"
  INITIAL ?
  POSITION 14
  MAX-WIDTH 15
  DECIMALS 0
  ORDER 5050
  MANDATORY
  FOREIGN-POS 5
  FOREIGN-NAME "ABC"
  FOREIGN-TYPE "NUMERIC"
  FIELD-MISC11 5
  FIELD-MISC12 0
  FIELD-MISC13 5
  FIELD-MISC14 10
  FIELD-MISC22 "N"
  FIELD-MISC24 ""
```

**Second Error:** Syntax error <unknown>:13652:2 extraneous input 'RECID-INDEX' ...
```
ADD INDEX "idx1" ON "XX123" 
  AREA "Schema Area"
  FOREIGN-NAME "idx1"
  RECID-INDEX "ux"
  INDEX-FIELD "fld1" ASCENDING 
  INDEX-FIELD "fld2" ASCENDING 
```

**Comment by jakejustus** [comment]
Thanks for having reported. This has been fixed in development (along with several other missing keywords in DF syntax).

**Comment by movedoa** [comment]
Should this be fixed in 1.6.2?
Im still getting these errors

**Comment by jakejustus** [comment]
I confirm the error is still there during the analysis. The parser itself has been updated, but the dependency on the parser has not been updated accordingly. Will publish a new version ASAP.

**Comment by jakejustus** [comment]
Fixed in 1.7.1
